code/_globalvars/traits.dm

```diff
@@ -57,7 +57,12 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_NOGERMS" = TRAIT_NOGERMS,
 		"TRAIT_NODECAY" = TRAIT_NODECAY,
 		"TRAIT_NOEXAMINE" = TRAIT_NOEXAMINE,
-		"TRAIT_NOPAIN" = TRAIT_NOPAIN)))
+		"TRAIT_NOPAIN" = TRAIT_NOPAIN
+		),
+		/obj/item = list(
+			"TRAIT_SHOW_WIRE_INFO" = TRAIT_SHOW_WIRE_INFO
+		)
```

Comment:
Nitpick: to keep it on the same indention as the `/mob = list(` above
```suggestion
	),
	/obj/item = list(
		"TRAIT_SHOW_WIRE_INFO" = TRAIT_SHOW_WIRE_INFO
	)
```

Reply:
Oh I thought I'd done that. Must've got confused by the long list.